library/network/src/lib/y2firewall/firewalld/relations.rb

```diff
@@ -3,21 +3,61 @@ class Firewalld
     # Extends the base class with metaprogramming methods which defines some
     # attributes common logic.
     module Relations
+      def enable_modifications_cache
+        class_eval("attr_accessor :modified")
```

Comment:
Are you sure you need a `class_eval` call here? I guess that calling to `attr_accessor` should work.

Reply:
Not needed